app/src/App.tsx

```diff
@@ -17,11 +24,53 @@ import {
   DatasetProvider,
   startMetadataLoad,
 } from "./utils/useDatasetStore";
-import { LinkWithStickyParams } from "./utils/urlutils";
+import {
+  LinkWithStickyParams,
+  EXPLORE_DATA_PAGE_LINK,
+  DATA_CATALOG_PAGE_LINK,
+  ABOUT_US_PAGE_LINK,
+} from "./utils/urlutils";
 import AboutUsPage from "./pages/AboutUsPage";
 
+const MOBILE_BREAKPOINT = 600;
+
+const PAGE_URL_TO_NAMES: Record<string, string> = {
+  "/": "Homepage",
+  [ABOUT_US_PAGE_LINK]: "About Us",
+  [DATA_CATALOG_PAGE_LINK]: "Data Sources & Methodology",
+  [EXPLORE_DATA_PAGE_LINK]: "Explore the Data",
+};
+
 startMetadataLoad();
 
+function MobileAppToolbar() {
+  const [open, setOpen] = useState(false);
+
+  function ListItemLink(props: any) {
+    return <ListItem button component="a" {...props} />;
+  }
+
+  return (
+    <Toolbar>
+      <IconButton onClick={() => setOpen(true)}>
+        <MenuIcon />
+      </IconButton>
+      <Drawer variant="persistent" anchor="left" open={open}>
+        <Button onClick={() => setOpen(false)}>
+          <ChevronLeftIcon />
+        </Button>
+        <List>
+          {Object.keys(PAGE_URL_TO_NAMES).map((pageUrl) => (
+            <ListItemLink href={pageUrl}>
+              <ListItemText primary={PAGE_URL_TO_NAMES[pageUrl]} />
+            </ListItemLink>
```

Comment:
do these list items need to have a key? Same below

Reply:
good call!